blocks/library/gallery/editor.scss

```diff
@@ -13,6 +12,26 @@
 	&.is-transient img {
 		@include loading_fade;
 	}
+
+	.blocks-rich-text__inline-toolbar {
```

Comment:
Should be `.block-rich-text__inline-toolbar` I think.

Reply:
Well, it's a typo in the `RichText` component it seems. :)

Reply:
(But this rule won't do anything.)

Reply:
Yes the name was updated 👍 The spacing was also updated because right our css rules changed and we don't require a big margin.

Reply:
Is it still needed at all? As long as the toolbar appears outside the black area, it's good. I didn't see it appearing inside the black area before.

Reply:
Yes in this layout the toolbar is not bad even without the rule. So I removed it.